robottelo/helpers.py

```diff
@@ -208,3 +214,55 @@ def remove_katello_ca(hostname=None):
             )
             if result.return_code != 0:
                 raise AssertionError('Failed to reset the rhsm.conf')
+
+
+def download_server_file(localpath, fileurl):
+    """Downloads a file from server using wget command.
+
+    :param str localpath: local path of file to write to from fileurl.
+    :param str fileurl: A full path to file on server.
+    :return: None
+    :raises: DownloadFileError: If file not downloaded
+
+    """
+    global _YAML_DOWNLOADED
```

Comment:
See comment above the reason of adding global variable. Any alternate suggestion is welcome.